addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -208,7 +205,11 @@ def _export_as_xml(self):
         :return: The XML content as str.
         '''
         template_values = self._prepare_fatturapa_export_values()
-        content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export')._render(template_values)
+        if(not self.env['account.edi.format']._l10n_it_is_simplified_document_type(template_values['document_type'])):
+            content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export')._render(template_values)
+        else:
+            content = self.env.ref('l10n_it_edi.account_invoice_it_simplified_FatturaPA_export')._render(template_values)
```

Comment:
We should check first if the module was updated by putting raise_if_not_found False and checking if this returns something or not.